src/components/Apps/AppsList/utils.tsx

```diff
@@ -53,30 +53,36 @@ const sortFunc = (
   return 0;
 };
 
+function computeShortTitle([key, catalog]: [string, IAppCatalog]): [
+  string,
+  IAppCatalog,
+  string
```

Comment:
These can also be labeled, to make them slightly less confusing

```suggestion
  key: string,
  catalog: IAppCatalog,
  shortTitle: string
```

Reply:
Thanks 👍